templates/travel/partials/footer/footer.snip.html

```diff
@@ -0,0 +1,110 @@
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+}}
+
+<!-- Footer -->
+<footer class="travel-footer overflow-hidden">
+  <section class="travel-footer-blog max-width-3 mx-auto px1 md-hide lg-hide">
+    <div class="flex md-mx2 my3">
+      <amp-img
+        class="rounded relative flex-none mx1"
+        width="100"
+        height="100"
+        src="../../../img/travel/blogpost-thumbnail.jpg"
+      ></amp-img>
+      <div class="flex-auto">
+        <div class="h3 bold line-height-2 mb1">From the blog</div>
+        <p class="travel-footer-blog-text my1">
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit amet dolor set. <a class="travel-link" href="#">Read more</a>
+        </p>
+      </div>
+    </div>
+  </section>
+
+  <div class="relative bg-black">
+
+    <!-- Angle -->
+    <div class="travel-footer-angle-block absolute xs-hide sm-hide"></div>
+    <div class="travel-footer-angle xs-hide sm-hide"></div>
+    <!--/ Angle -->
+
+    <!-- Right column -->
+    <div class="travel-footer-right-column">
+      <div class="max-width-3 mx-auto px1 md-px2">
+        <h4 class="travel-footer-right-column-heading travel-spacing-none h2 mb3 blue">Want travel deals<br>straight to your inbox?</h4>
+        <div class="h4 bold mb1">Sign up to our newsletter</div>
+      </div>
+      <div class="relative">
+        <div class="travel-footer-input-bg bg-black absolute right-0 left-0 md-hide lg-hide"></div>
+        <div class="max-width-3 mx-auto px1 md-px2 relative">
+          <div class="travel-input-group flex items-center col-12 rounded travel-shadow">
+            <input
+              class="travel-input travel-input-big block col-12 flex-auto rounded-left"
+              type="text"
+              name="email"
+              placeholder="Enter your email address..."
+            >
+            <span class="travel-input-group-sep travel-border-gray relative z1 block"></span>
+            <button
+              type="button"
+              class="travel-link travel-input-big nowrap rounded-right"
+            >
+              Sign up now
+            </button>
+          </div>
+        </div>
+      </div>
+    </div>
+    <!--/ Right column -->
+
+    <!-- Left column -->
+    <div class="max-width-3 mx-auto px1 md-px2">
+      <div class="clearfix mxn3 pt3 md-pt4">
+        <div class="col-12 md-col-6 px3 clearfix">
+          <div class="col col-2">
+            <a href="travel.amp" class="inline-block h2 line-height-1 circle white">
+              {{> ../../../../img/icon/logo.svg}}
+            </a>
+          </div>
+          <div class="col col-5">
+            <h4 class="travel-spacing-none mb2 h3 gray bold">Company</h4>
+            <div class="line-height-4 mb4">
+              <div><a href="#" class="travel-link gray">About</a></div>
```

Comment:
remove all the wrapping divs  and make the a inline-block? or are they actually needed?